.github/workflows/main.yml

```diff
@@ -80,6 +80,12 @@ jobs:
     steps:
     - uses: actions/checkout@v1
 
+    - name: free disk space
+      run: |
+          sudo apt clean
```

Comment:
I thought you would do this in the Dockerfile. That would decrease the size of the image but not make the CI build slower

Reply:
I do not think I understand the difference between the two choices, in terms of why one makes the CI build slower, and not the other.